### ISSUE TITLE
Deprecate the status page during peak traffic

### ISSUE THREAD

**Issue by pixernot** [KeypDemoCorp/awesome-api-backend#372]
Accessibility should be considered when implementing this.

**Comment by pixernot** [comment]
I'll take the first pass and open a draft PR for feedback.
